apps/backend/src/interceptors/api-keys-allowed-interceptor.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  CallHandler,
+  ExecutionContext,
+  ForbiddenException,
+  Injectable,
+  NestInterceptor
+} from '@nestjs/common';
+import {Observable} from 'rxjs';
+import {ConfigService} from '../config/config.service';
+
+@Injectable()
+export class ApiKeysAllowedInterceptor implements NestInterceptor {
+  private readonly configService: ConfigService;
+  constructor(configService: ConfigService) {
+    this.configService = configService;
+  }
+
+  intercept(_context: ExecutionContext, next: CallHandler): Observable<void> {
```

Comment:
With my other suggestions this would effectively change to a server error if the new API secret env is not set.

Reply:
I actually think this file may be removed